autobuild/startup_bridged.toml

```diff
@@ -0,0 +1,6 @@
+[startup.bridged]
+name = "core.system"
+
+[startup.bridged.args]
+name = "/bin/bridged"
+args = ["--rpc-addr={BRIDGED_RPC_ADDR}", "--ethnetwork={ETH_NETWORK}", "--ethport={ETH_PORT}", "--account-json={ETH_ACCOUNT_JSON}", "--account-password={ETH_ACCOUNT_PASSWORD}"]
```

Comment:
Are these flags optional or always present? The `--account-json` will likely only be used once (if at all), while the ethnetwork is set by default by the bridge itself if not given, depending on which tft network is joined, and it seems unlikely that a deployment via a template will use another value

Reply:
those are always present, maybe in code we support empty values for account-json and the ethnetwork to support that behavior?

Reply:
I see. In that case, please remove the `--ethnetwork` flag and replace it with the regular `--network` flag with the tfchain network (like in the blockcreator), as the daemon sets a default eth network to join based on the tft network